cmd/kubeless/autoscale.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2016 Skippbox, Ltd.
```

Comment:
Wrong copyright owner (ditto other files)

Reply:
I will send a PR to update license header to all files 👍

Reply:
I just added PR for this https://github.com/kubeless/kubeless/pull/273